pallets/asset-manager/src/lib.rs

```diff
@@ -551,6 +563,25 @@ pub mod pallet {
             });
             Ok(())
         }
+
+        /// Set min xcm fee for asset/s on their reserve chain.
+        ///
+        /// * `origin`: Caller of this extrinsic, the access control is specified by `ForceOrigin`.
+        /// * `filtered_location`: Multilocation to be filtered.
+        #[pallet::call_index(6)]
+        #[pallet::weight(T::WeightInfo::update_outgoing_filtered_assets())]
+        #[transactional]
+        pub fn update_outgoing_filtered_assets(
+            origin: OriginFor<T>,
+            filtered_location: T::Location,
+        ) -> DispatchResult {
+            T::ModifierOrigin::ensure_origin(origin)?;
+            FilteredOutgoingAssetLocations::<T>::insert(filtered_location.clone().into(), ());
```

Comment:
I think we might check the related asset is created or not first.

Reply:
why though ?

Reply:
If the multilocation we don't support, why we add it to this storage?

Reply:
Yeah it doesn't really make sense but could it lead to something bad  ?
Otherwise it's just an extra read that doesn't seem necessary